our-brothers-web/src/app/agenda/page/agenda-page.component.scss

```diff
@@ -141,4 +143,31 @@
       }
     }
   }
+
+  .pages-number-container {
```

Comment:
Can we remove the default `background: transparent;`?

Reply:
I tried it, and the background is changed to grey.
so we must leave it here